cli_reference/extend_cli.adoc

```diff
@@ -0,0 +1,218 @@
+[[cli-reference-extend-cli]]
+= Extend the CLI
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[cli-plugins-overview]]
+== Overview
+
+This topic reviews how to install and write extensions for the CLI. Usually
+called _plug-ins_ or _binary extensions_, this feature allows you to extend the
+default set of `oc` commands available and, therefore, allows you to perform new
+tasks.
+
+A plug-in is a set of files: typically at least one *_plugin.yaml_* descriptor
+and one or more binary, script, or assets files.
+
+CLI plug-ins are currently only available under the `oc plugin` subcommand.
+
+ifndef::openshift-dedicated[]
```

Comment:
There's an 'n' in `ifdef` here. Does it build correctly?

Reply:
@bmcelvee Yeah, ifndef is intentional here. It means to include the content in all repos except dedicated.